docs/install/setting_up_cluster_on_amazon_web_services.md

```diff
@@ -242,42 +247,50 @@ This sections covers the following:
 *	[Destroying the Cluster](#ec2destroycluster)
 *	[Starting Cluster with Apache Zeppelin](#ec2startclustrezeppelin)
 *	[More Options](#ec2moreoption)
-<a id="ecsspecifyprop"></a>
-#### Specifying properties
 
-If you want to configure each of the locator, lead, or server with specific properties, you can do so by specifying them in files named **locators**, **leads** or **servers**, respectively and placing these under **aws/ec2/deploy/home/ec2-user/snappydata/**. Refer to [this SnappyData documentation page](../configuring_cluster/configuring_cluster.md#configuration-files) for example on how to configure these configuration files.</br>
-This is similar to how one would provide properties to SnappyData cluster nodes while launching it using the `sbin/snappy-start-all.sh` script.</br>
-The important difference here is that, instead of the host names of the locator, lead or store, you have to write {{LOCATOR_N}}, {{LEAD_N}} or {{SERVER_N}} in these files, respectively. N stands for Nth locator, lead or server. The script replaces these with the actual host name of the members when they are launched.
+<a id="ec2custombuild"></a>
+#### Using custom build
 
-The sample configuration files for a cluster with 2 locators, 1 lead and 2 stores are given below:
+This script by default uses the SnappyData build available on the GitHub releases page to launch the cluster.
+To select a version of the build available on GitHub, use option `--snappydata-version`.
 
-*locators*
+You can also provide your own SnappyData build to the script to launch the cluster.
```

Comment:
I would specifically tell the reader that this is the option you should use if you want to use the enterprise version of the product. Download enterprise product from 'link'...

Reply:
Have updated the description.